### DISCUSSION TITLE
Website not working for active sessions after each Vercel deployment

### DISCUSSION THREAD

**Discussion by testvinder** [sveltejs/kit#12104] [Q&A] [answered]
I'm using latest versions of Sveltekit and the Vercel adapter.
"@sveltejs/adapter-vercel": "5.2.0",
"@sveltejs/kit": "2.5.5",
"svelte": "4.2.12"

Every time I deploy a new version to prod in Vercel the website stops working for active sessions. The browser attempts to load old modules. If I open the website in incognito or in a different browser the problem is not there.

The error shown in the browser console is "Uncaught (in promise) TypeError: error loading dynamically imported module: " In the Vercel logs I can also see 404s for various js files such as "/_app/immutable/chunks/store.CLkuf-ir.js".

I'm not using Cloudflare or any other caching layer. I have Skew Protection enabled.

Have anyone faced a similar issue or now what might be causing it?

**Comment by teemingc** [comment]
Hi Mads, could you share a minimal reproduction? You're not alone in this issue and it would be a big help if you shared more information so that we can diagnose the root cause of this. Related to https://github.com/sveltejs/kit/issues/9089

**Reply by testvinder** [reply, marked as answer]
Hi Tee,

Unfortunately, it's difficult to reproduce as the error only happens in production and only for users with an active session during deployment.

However, the issue seems to have disappeared now after disabling Skew Protection.

**Reply by Chizaruu** [reply]
> Hi Tee,
> 
> Unfortunately, it's difficult to reproduce as the error only happens in production and only for users with an active session during deployment.
> 
> However, the issue seems to have disappeared now after disabling Skew Protection.

If only I had seen this 9 hours ago, it would've saved me the headache of debugging why users weren't using the new Server side code.
